From 4a6b8c04e863f9359844a8dcc74101a50d176f11 Mon Sep 17 00:00:00 2001
From: John Zhuge <jzhuge@cloudera.com>
Date: Thu, 11 Aug 2016 13:36:11 -0700
Subject: [PATCH 1719/2748] HDFS-8633. Fix setting of
 dfs.datanode.readahead.bytes in hdfs-default.xml
 to match DFSConfigKeys. Contributed by Ray
 Chiang.

(cherry picked from commit b42f1ec13d0a837f188cf424a8dd177720478cc9)
(cherry picked from commit 8c03a079df78fd094b27025a6f700e4d99428ea4)

Change-Id: I253e742522e74459974bfd9d8a5858c6026ed60b
---
 .../src/main/resources/hdfs-default.xml            |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml b/hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml
index cd6d60e..32c8536 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml
@@ -1093,7 +1093,7 @@
 
 <property>
   <name>dfs.datanode.readahead.bytes</name>
-  <value>4193404</value>
+  <value>4194304</value>
   <description>
         While reading block files, if the Hadoop native libraries are available,
         the datanode can use the posix_fadvise system call to explicitly
-- 
1.7.9.5

